htdocs/mrp/class/mo.class.php

```diff
@@ -763,7 +763,10 @@ public function createProduction(User $user, $notrigger = 0)
 							if ($line->qty_frozen) {
 								$moline->qty = $line->qty; // Qty to consume does not depends on quantity to produce
 							} else {
-								$moline->qty = (float) price2num(($line->qty / (!empty($bom->qty) ? $bom->qty : 1)) * $this->qty / (!empty($line->efficiency) ? $line->efficiency : 1), 'MS'); // Calculate with Qty to produce and  more presition
+								$efficiency = !empty($line->efficiency) ? $line->efficiency : 1;
+								$lossfactor = max(0, 2 - $efficiency); // Factor is 1 when efficiency is 1, >1 when efficiency <1
```

Comment:
If efficiency is 0.1 (10%), it means we must have a quantity of 10 if we cant to get 1.
This is what we got with old code.

With new code, if we have an efficicency of 10% only, the planned value to consume to get 1 is 1.9 ???

Before, making a change on this, thanks to introduce a phpunit test with sample of value with expected result that show that result is wrong.